patches/api/0373-Collision-API.patch

```diff
@@ -3,7 +3,63 @@ From: Owen1212055 <23108066+Owen1212055@users.noreply.github.com>
 Date: Wed, 6 Oct 2021 20:10:36 -0400
 Subject: [PATCH] Collision API
 
+Co-authored-by: Tamion <70228790+notTamion@users.noreply.github.com>
 
+diff --git a/src/main/java/io/papermc/paper/event/entity/EntityCollideWithEntityEvent.java b/src/main/java/io/papermc/paper/event/entity/EntityCollideWithEntityEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..a37d7f50dc09b12c8ae56a3180f146b4d77ac8d7
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/event/entity/EntityCollideWithEntityEvent.java
+@@ -0,0 +1,49 @@
++package io.papermc.paper.event.entity;
++
++import org.bukkit.entity.Entity;
++import org.bukkit.event.Cancellable;
++import org.bukkit.event.Event;
++import org.bukkit.event.HandlerList;
++import org.jetbrains.annotations.NotNull;
++import java.util.HashSet;
++import java.util.Set;
++
++public class EntityCollideWithEntityEvent extends Event implements Cancellable {
++
++    private static final HandlerList handlers = new HandlerList();
```

Comment:
`HANDLER_LIST`. Best to use the standards for naming static finals (no matter that upstream doesn't do that)

Reply:
Isn't this naming style is used only for constants? `HandlerList` is changing when (un)registering listeners, so it should not be considered a constant.

Reply:
convention within bukkit is to call it `HANDLER_LIST`